### PR TITLE
Fix init reader bug

### PR DESCRIPTION
**Pull request by hatianzhang**
# Description

init llamaindex miss a bunch of readers
Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama_index/__init__.py [hunk 1]
```diff
@@ -64,7 +64,34 @@
...
+    SimpleMongoReader,
+    SimpleWebPageReader,
+    SlackReader,
+    StringIterableReader,
+    TrafilaturaWebReader,
+    TwitterTweetReader,
+    WeaviateReader,
+    WikipediaReader,
+    download_loader,
+)
```
Comment (logan-markewich) [inline llama_index/__init__.py:94]: Oh, we actually don't want to import these here.

Was there a notebook that was importing from here? we should update it

Comment (hatianzhang) [reply llama_index/__init__.py:94]: yes, `from llama_index import SummaryIndex, RssReader`

Comment (logan-markewich) [reply llama_index/__init__.py:94]: we should update that example instead 👍🏻

Comment (hatianzhang) [reply llama_index/__init__.py:94]: @logan-markewich there are a bunch of readers defined in this file, should we remove them from ```
__all__ = [***]```

Comment (logan-markewich) [reply llama_index/__init__.py:94]: Yea, only simple directory reader and download_loader should be used and exported here

**Review by logan-markewich — APPROVED** [review]

**Pull request merged**